code/modules/antagonists/eldritch_cult/eldritch_book.dm

```diff
@@ -0,0 +1,136 @@
+/obj/item/forbidden_book
+	name = "Codex Cicatrix"
+	desc = "Book describing the secrets of the veil."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "book"
+	var/mob/living/last_user
+	var/charge = 1
+	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava))
+
+/obj/item/forbidden_book/examine(mob/user)
+	. = ..()
+	if(!user.mind.has_antag_datum(/datum/antagonist/ecult))
+		return
+	. += "The Tome holds [charge] charges."
+
+/obj/item/forbidden_book/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!proximity_flag)
+		return
+	if(istype(target,/obj/effect/eldritch))
+		remove_rune(target,user)
+	if(istype(target,/obj/effect/reality_smash))
+		get_power_from_influence(target,user)
+	if(istype(target,/turf/open))
+		draw_rune(target,user)
+	return
+
+/obj/item/forbidden_book/proc/get_power_from_influence(atom/target, mob/user)
+	var/obj/effect/reality_smash/RS = target
+	if(do_after(user,10 SECONDS,FALSE,RS))
```

Comment:
this will succeed only once if you spam-click on an item, right?

Reply:
Yes, coz it destroys the target afterwards, or at least i think thats how it works, ill check it tommorow